tests/unit/checkpoint/test_latest_checkpoint.py

```diff
@@ -5,10 +5,15 @@
 
 import deepspeed
 
+import pytest
 from unit.common import DistributedTest
 from unit.simple_model import *
 
 from unit.checkpoint.common import checkpoint_correctness_verification
+from deepspeed.ops.op_builder import FusedAdamBuilder
+
+if not deepspeed.ops.__compatible_ops__[FusedAdamBuilder.NAME]:
```

Comment:
I think [torch Adam](https://pytorch.org/docs/stable/_modules/torch/optim/adam.html#Adam) should work in CPU accelerator environment.  Can you please confirm if I am correct?  If so, then it will be better to test torch Adam for CPU accelerator environment rather than disable this test entirely. 

Please see [here](https://github.com/microsoft/DeepSpeed/blob/691d246e0216a5c209eff10a7a6ee8c78eed521e/tests/unit/ops/adam/test_adamw.py#L53) for an example of how to enable torch.Adam.

Please let me know if this change is possible. Thanks!

Reply:
Hi, we will submit another PR after we finish FusedAdamBuilder. Thanks~